src/control-plane/control-plane-api.ts

```diff
@@ -16,11 +15,25 @@ import { IAuth } from './auth/auth-interface';
 import { Services } from './services';
 import { addTemplateTag, conditionallyAddScope } from '../utils';
 
+export interface APICorsConfig {
```

Comment:
Not that it's bad, necessarily, buy why not just use [the interface](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigatewayv2.CorsPreflightOptions.html) that's already in CDK?

Reply:
i was trying to make it more generic so it can be used even in cases where we are not using API GW v2, but I guess that's not really working because I'm still using this in the interface:

```
  readonly allowMethods?: apigatewayV2.CorsHttpMethod[];
```

Should i just use `cdk.aws_apigatewayv2.CorsPreflightOptions` instead?

Reply:
Personally, I would. If the interface ever changed we'd have to change ours to match anyway. Doesn't seem like re-exporting buys us much. Just my 2¢.

Reply:
updated!